src/CorsMiddleware.php

```diff
@@ -269,11 +277,7 @@ private function origin(array $origin): void
     private function methods($methods): void
     {
         if (is_callable($methods)) {
-            if ($methods instanceof Closure) {
-                $this->options["methods"] = $methods->bindTo($this);
-            } else {
-                $this->options["methods"] = $methods;
-            }
+            $this->options["methods"] = $methods instanceof Closure ? $methods->bindTo($this) : $methods;
```

Comment:
Please let me know what you think about the usage of ternary operators :smile:

Reply:
It depends on the case. In this case the code is harder to read and the resulting line is unnecessarily long. These days I actually prefer something like:

```php
$this->options["error"] = $error;
if ($error instanceof Closure) {
    $this->options["error"] = $error->bindTo($this);
}
```

or actually:

```php
$this->error = $error;
if ($error instanceof Closure) {
    $this->error = $error->bindTo($this);
}
```

In any case I would disable the `SimplifyIfElseToTernaryRector` rule for now.

Reply:
> It depends on the case. In this case the code is harder to read and the resulting line is unnecessarily long. 

That's exactly how I see it. And most of these one-liners are forgotten in the tests because it doesn't show up in the coverage.

> In any case I would disable the `SimplifyIfElseToTernaryRector` rule for now.

I agree.